crd-convert/src/main/java/io/strimzi/kafka/crd/convert/cli/ConvertCommand.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.crd.convert.cli;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.fabric8.kubernetes.client.CustomResource;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.dsl.MixedOperation;
+import io.strimzi.api.annotations.ApiVersion;
+import io.strimzi.api.kafka.model.Kafka;
+import io.strimzi.api.kafka.model.KafkaBridge;
+import io.strimzi.api.kafka.model.KafkaConnect;
+import io.strimzi.api.kafka.model.KafkaConnectS2I;
+import io.strimzi.api.kafka.model.KafkaMirrorMaker;
+import io.strimzi.api.kafka.model.KafkaMirrorMaker2;
+import io.strimzi.api.kafka.model.KafkaRebalance;
+import io.strimzi.api.kafka.model.KafkaTopic;
+import io.strimzi.api.kafka.model.KafkaUser;
+import io.strimzi.kafka.crd.convert.converter.Converter;
+import io.strimzi.kafka.crd.convert.converter.KafkaBridgeConverter;
+import io.strimzi.kafka.crd.convert.converter.KafkaConnectConverter;
+import io.strimzi.kafka.crd.convert.converter.KafkaConnectS2IConverter;
+import io.strimzi.kafka.crd.convert.converter.KafkaConverter;
+import io.strimzi.kafka.crd.convert.converter.KafkaMirrorMaker2Converter;
+import io.strimzi.kafka.crd.convert.converter.KafkaMirrorMakerConverter;
+import io.strimzi.kafka.crd.convert.converter.KafkaRebalanceConverter;
+import io.strimzi.kafka.crd.convert.converter.KafkaTopicConverter;
+import io.strimzi.kafka.crd.convert.converter.KafkaUserConverter;
+import picocli.CommandLine;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiFunction;
+
+@SuppressWarnings("deprecation")
+@CommandLine.Command(name = "convert", aliases = {"c"}, description = "Convert CRDs")
+public class ConvertCommand extends JYCommand {
+    @CommandLine.Option(
+        names = {"-fv", "--from-version"},
+        description = "From K8s ApiVersion - required when used with Kube API",
+        completionCandidates = Versions.class
```

Comment:
This is needed in the Crds class, otherwise default version is used, which will probably be latest - v1beta2.
So you cannot direct the tool / k8s client to older versions w/o this option.

Reply:
I guess I understand why you need it internally. But is it really something the customer needs to understand?